src/types.ts

```diff
@@ -0,0 +1,16 @@
+export class ExporterPosition {
+  blockNumber: number;
+  primaryKey: number;
+
+  constructor(blockNumber: number, primaryKey: number) {
+    this.blockNumber = blockNumber;
+    this.primaryKey = primaryKey;
+  }
+}
+
+// This interface is modeled agains the jayson Client that we actually use
```

Comment:
Wrap work with Jaysong client through an interface. Also serves to help testing.